src/view.ts

```diff
@@ -269,6 +269,7 @@ export default class CalendarView extends ItemView {
       // File doesn't exist
       tryToCreateWeeklyNote(startOfWeek, inNewSplit, this.settings, (file) => {
         activeFile.setFile(file);
+        workspace.setActiveLeaf(leaf, true, true)
```

Comment:
This doesn't look right. `leaf` doesn't exist in this context.

Reply:
my bad, was trying to do this
https://github.com/liamcain/obsidian-calendar-plugin/pull/178/commits/8dbce6996b26ea5c9dc7e98f317f6e3fac219f8d